result/metrics.yaml

```diff
@@ -107,10 +112,12 @@ components:
               x-field-uid: 10
             dhcpv4_server:
               x-field-uid: 11
-            dhcpv6_client:
+            ospfv2:
```

Comment:
Dont insert in middle , looks to be breaking backward compatibity ? How come builder did not catch this ?  Probably separate issue needs to be raised.

Reply:
@SuryyaKrJana : This is still there and seems like a risky issue.